indico/web/client/js/react/components/principals/TrackACLField.jsx

```diff
@@ -0,0 +1,52 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+
+import {ACLField} from 'indico/react/components';
+import {FavoritesProvider} from 'indico/react/hooks';
+import {PermissionManager} from 'indico/react/components/principals/util';
+
+export default function TrackACLField({value, permissionInfo, onChange}) {
+  const [currentValue, setCurrentValue] = useState(value);
+  const permissionManager = new PermissionManager(permissionInfo.tree, permissionInfo.default);
+  return (
+    <FavoritesProvider>
```

Comment:
You don't need this, it's just a wrapper for class components which can't use hooks. Use `const favoriteUsersController = useFavoriteUsers();` directly.